packages/workspace-resolver/package.json

```diff
@@ -4,8 +4,8 @@
   "license": "Apache-2.0",
   "main": "src/index.ts",
   "dependencies": {
+    "@esm2cjs/globby": "13.1.4",
```

Comment:
What was the need to switch to the forked version of `globby`?

Reply:
OG globby is now a pure ESM package that we can't import